src/test/java/org/apache/logging/log4j/ext/kusto/log4j2.component.properties

```diff
@@ -0,0 +1 @@
+log4j2.configurationFactory=org.apache.logging.log4j.kusto.KustoLog4jTestConfigurationFactory
```

Comment:
```suggestion
log4j2.configurationFactory=org.apache.logging.log4j.kusto.KustoLog4jTestConfigurationFactory

```

Reply:
@asaharn some of the new line requirements may not be needed as they are used within tests